gulpfile.js

```diff
@@ -0,0 +1,10 @@
+var gulp = require('gulp');
+var replace = require('gulp-replace');
+var packageJson = require('./package.json');
+var bowerJson = require('./bower.json');
+
+gulp.task('propagate-version', function() {
+  gulp.src(['./bower.json', './juicy-filedrop.html'])
+    .pipe(replace(bowerJson.version, packageJson.version))
```

Comment:
Maybe as it is gulp, not grunt, and you don't like such long RegExp, we could write more code, but more explicit one, to pick just the applicable version an replace it?